policies/TrademarkPolicy.md

```diff
@@ -0,0 +1,215 @@
+# Trademark Policy
+
+#### Last modified 2022-11-22
+
+
+
+# Purpose
+
+OpenSSL is committed to promoting the use of its open source software. While 
+open source software is generally free to download and modify, the use of
+open source software does not include the right to OpenSSL trademarks.
+
+The Trademark Policy (The Policy) aims to protect and ensure consistent usage 
+of the OpenSSL trademarks and to clarify how the OpenSSL trademarks may be 
+used.
+
+As a part of this process, the OpenSSL trademark is a registered United States 
+trademark of the OpenSSL Software Foundation. 
+For registration outside the United States, we have filed a [Madrid Protocol](http://en.wikipedia.org/wiki/Madrid_system) 
+application to extend the protection in the European Union, China and Japan.
+
+
+The Policy aims to:
+
+1.  to encourage appropriate, authorised use of the OpenSSL trademarks;
+2.  to clarify proper usage of OpenSSL trademarks by third parties;
+3.  to prevent misuse of OpenSSL trademarks that can confuse or mislead
+    users with respect to OpenSSL or its affiliates.
+
+Please note that it is not the goal of The Policy to limit commercial
+activity around OpenSSL. We encourage businesses to work on OpenSSL
+while being compliant with The Policy.
+
+
+# Scope
+
+The Policy encompasses all marks, in word and logo form including but not 
+limited to, OpenSSL name and derivatives thereof, software names, project 
+names, logos and styles collectively referred to as "Our Trademarks". 
+
+The Policy applies to all persons who use or intend to use Our Trademarks.
+
+
+# Principles
+
+* Our Trademarks can only be used in line with The Policy, including but not 
+limited to; use in software, advertising or on websites.
+* Our Trademarks can only be used to identify OpenSSL, its products and 
+services in a way that is accurate, not misleading or disparaging of 
```

Comment:
I'm a little unsure about the "disparaging" here. Does this stifle valid criticism of OpenSSL?